### ISSUE TITLE
[Feedback] Feedback from guidepup new user  

### ISSUE THREAD

**Issue by vasylshlapaks** [guidepup/guidepup#29]
First of all I want to say thank you for such a great tool and documentation. 
I was able to setup everything and work with VoiceOver. Examples of usage with Playwright and CI examples are very useful as well. 

One thing that I think can be improved - documentation for API. It was sometimes easier to explore the classes and src files to understand methods and their options. From one side api documentation is deep enough but feels like it's lack of examples, especially for proper usage for things like  `await voiceOver.perform(voiceOver.keyboard.commands.findNextDifferentItem)` (where you provide complex usage of few apis) 
If you have some time in future to add examples like Playwright api docs have for their apis for example, it would make usage even more smoother.

Again thanks for your work!

**Comment by cmorten** [comment]
@vasylshlapaks thanks for the feedback!

More examples sounds like a plan, will see what can do 😊

**Comment by cmorten** [comment]
@vasylshlapaks I've gone through and added examples for the majority of the docs site on <https://guidepup.dev>.

I'd appreciate it if you could give a sense whether what we have now is the level of docs/examples you might want?

**Comment by vasylshlapaks** [comment]
@cmorten I checked it and looks great, thank you
